egs/swbd/s5c/local/nnet3/tuning/run_tdnn_c.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+# c is as a, but with 1 more extra layer which gives 0.5-0.6% absolute 
+# improvement.
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+stage=0
+affix=
+train_stage=-10
+has_fisher=true
+speed_perturb=true
+common_egs_dir=
+reporting_email=
+remove_egs=true
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+suffix=
+if [ "$speed_perturb" == "true" ]; then
+  suffix=_sp
+fi
+dir=exp/nnet3/tdnn_a
+dir=$dir${affix:+_$affix}
+dir=${dir}$suffix
+train_set=train_nodup$suffix
+ali_dir=exp/tri4_ali_nodup$suffix
+
+local/nnet3/run_ivector_common.sh --stage $stage \
+	--speed-perturb $speed_perturb || exit 1;
+
+if [ $stage -le 9 ]; then
+  echo "$0: creating neural net configs";
+
+  # create the config files for nnet initialization
+  steps/nnet3/tdnn/make_configs.py  \
+    --feat-dir data/${train_set}_hires \
+    --ivector-dir exp/nnet3/ivectors_${train_set} \
+    --ali-dir $ali_dir \
+    --relu-dim 1024 \
+    --splice-indexes "-2,-1,0,1,2 -1,2 -3,3 -3,3 -7,2 0"  \
```

Comment:
Oops, I realized there is one more thing that needs to be done to get this working on non-chain nnet3 the same way as the baseline setup was working.  It relates to the presoftmax-prior-scale.  I *think* you can write:
```
output-layer name=output input=tdnn7 dim=$num_targets max-change=1.5 presoftmax-prior-scale-file=$dir/configs/presoftmax_prior_scale.vec
```
and it should work correctly with the current script.  At least that's the way it was designed to work.
I don't know if this has ever been tested.
@vijayaditya might know.